pom.xml

```diff
@@ -57,10 +57,26 @@
 		<spring-cloud-circuitbreaker.version>3.0.0-SNAPSHOT</spring-cloud-circuitbreaker.version>
 		<spring-cloud-commons.version>4.0.0-SNAPSHOT</spring-cloud-commons.version>
 		<testcontainers.version>1.17.3</testcontainers.version>
+		<micrometer.version>1.10.0-SNAPSHOT</micrometer.version>
```

Comment:
What versions does boot manage?

Reply:
`1.10.0-M3` right now but I have an open PR for `M4`.
They also want to move to `SNAPSHOT` ~in September I guess.